src/index.css

```diff
@@ -2,4 +2,251 @@ body {
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.log {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: var(--primary-font);
+  font-size: 1rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.menu-hamberger > div {
+  height: 3px;
+  width: 2rem;
+  background: #0290ff;
+  margin: 5px 0;
+  border-radius: 8px;
+}
+
+.profile a span {
+  display: none;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+form input {
+  height: 1.6rem;
+  margin: 1rem 1rem 0 0;
+  padding: 0.813rem 0.813rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  font-family: var(--primary-font);
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: #c4c4c4;
+}
+
+form button {
+  margin: 1.188rem 0 0 0;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+  text-transform: uppercase;
+  width: 11.5rem;
+  font-family: var(--secondary-font);
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+}
+
+.new-book {
+  margin: 0 0 2.375rem;
+  padding: 1rem 1rem 1rem 1rem;
+}
+
+.new-book h2 {
+  height: 1.5rem;
+  margin: 1.813rem 0.75rem 1.188rem 0;
+  font-family: var(--primary-font);
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+@media screen and (min-width: 768px) {
+  header {
+    width: 87.5rem;
+    height: 3rem;
+    margin: 0 0 2.375rem;
+    padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+    background-color: #fff;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  nav {
+    display: flex;
+    gap: 2rem;
+  }
+
+  nav a {
+    width: 3.688rem;
+    height: 1rem;
+    margin: 1.125rem 2.563rem 0.688rem 0;
+    font-family: var(--primary-font);
+    font-size: 0.813rem;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 1.9px;
+    color: #121212;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  .log {
+    width: 15rem;
+    height: 2.313rem;
+    margin: 0.375rem 2.938rem 0.125rem 0;
+    font-family: var(--primary-font);
+    font-size: 1.875rem;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    color: #0290ff;
+  }
```

Comment:
[OPTIONAL] One of the Bad CSS Practices is using the Same Rule Repetitively. I noticed that you use repetitive rules for each element. If you want to create a rule for multiple elements, use a class attribute instead.
For instance, instead of repeating font params (font-weight, font-style...) for each class:
```css
nav a {
   font-weight: normal;
   font-stretch: normal;
   font-style: normal;
   line-height: normal;
}

.log {
   font-weight: normal;
   font-stretch: normal;
   font-style: normal;
   line-height: normal;
}
```
Use a class instead. Ex:
```css
.font-params {
   font-weight: normal;
   font-stretch: normal;
   font-style: normal;
   line-height: normal;
}
```